README.md

```diff
@@ -3,8 +3,7 @@ Customizable bot for roguelike deck-building game [Slay the Spire](https://store
 
 ## FAQ
 ### Does it win a lot? Is it super smart?
-Depends on which character and strategy is in use. We've got a few different ones! Our current best is Watcher, and there we've got a 62% chance of beating the Act 3 boss.
-We'll re-test all characters and publish up-to-date win rates soon.
+Depends on which character and strategy is in use. We've got a few different ones! You can see our most current [winrates here](docs/winrates.md).
```

Comment:
Think we should still have an example here so they don't need to click somewhere else to get an idea of how good the bot is. Maybe just Watcher 52% then.

Reply:
Sure!